app/controllers/schools_controller.rb

```diff
@@ -23,21 +22,4 @@ def create
   def school_parameters
     params.require(:school).permit(:name, :opened_at, :school_type)
   end
-
-  def validate_opened_date
```

Comment:
Sooo, do we do any date validation? It's removed here, but I don't see it added anywhere?

Reply:
Doesn't look like it, my validation tests fail on this branch

Reply:
The model validates presence of opened_at and rails automatically checks that it's a date. Personally, I'd just change the spec to check that the bad data results in an exception.

Reply:
But we don't want an unhandled exception. We want a validation message on the form, right? I don't think this does that...